config.js

```diff
@@ -73,6 +73,7 @@ Config.prototype._seed = function _seed(seed) {
     seedOrDefault('gossipLogLevel', LoggingLevels.error);
     seedOrDefault('joinLogLevel', LoggingLevels.warn);
     seedOrDefault('membershipLogLevel', LoggingLevels.error);
+    seedOrDefault('ringLogLevel', LoggingLevels.debug);
```

Comment:
No. Good point. I'm using this branch to double as a PR and as the ringpopd test branch. Thanks for calling this out. I'll change back to 'error' level before merging.